continuous-integration/python3.sh

```diff
@@ -6,3 +6,5 @@ python setup.py install
 python -c "import sunpy.data"
 python -c "import sunpy.data; sunpy.data.download_sample_data()"
 python -c "import sunpy.data.sample"
+
+python setup.py test -P time
```

Comment:
sometimes I've found that if a script does not ends in a new line such line does not work...